Source/scrollrt.cpp

```diff
@@ -284,7 +284,7 @@ void DrawMissile(const Surface &out, int x, int y, int sx, int sy, bool pre)
 	if ((dFlags[x][y] & BFLAG_MISSILE) == 0)
 		return;
 
-	if (dMissile[x][y] != -1) {
+	if (dMissile[x][y] > 0) {

```

Comment:
What does `0` represent? Why do we treat both -1 and 0 the same? Shouldn't it really only be `-1`? Would it make more sense to use `-1` in the places where `0` is used?

Reply:
I see most of the info is in the linked issue. Should've read that before asking.

Reply:
no problem!